test/Java8andUp/src_80/org/openj9/test/java/lang/ref/Test_Reference.java

```diff
@@ -1,7 +1,7 @@
 package org.openj9.test.java.lang.ref;
 
 /*******************************************************************************
- * Copyright (c) 1998, 2017 IBM Corp. and others
+ * Copyright (c) 1998, 2018 IBM Corp. and others
```

Comment:
The copyright date update should not be required when moving files. But its not harmful...